src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -1012,6 +1012,99 @@ def test_export_template_spec(self, resource_group, resource_group_location):
 
 
 
+class DeploymentTestsWithQueryString(LiveScenarioTest):
```

Comment:
@zhoxing-ms @Juliehzl The query string tests are live tests because --query-string is an SAS token which cannot be included in PR (cred scan would fail).